test/Microsoft.AspNetCore.Server.KestrelTests/FrameResponseHeadersTests.cs

```diff
@@ -175,16 +175,6 @@ public void ThrowsWhenSettingContentLengthToNonNumericValue(string contentLength
 
         [Theory]
         [MemberData(nameof(BadContentLengths))]
-        public void ThrowsWhenSettingRawContentLengthToNonNumericValue(string contentLength)
-        {
-            var headers = new FrameResponseHeaders();
-
-            var exception = Assert.Throws<InvalidOperationException>(() => headers.SetRawContentLength(contentLength, Encoding.ASCII.GetBytes(contentLength)));
-            Assert.Equal($"Invalid Content-Length: \"{contentLength}\". Value must be a positive integral number.", exception.Message);
-        }
-
-        [Theory]
-        [MemberData(nameof(BadContentLengths))]
         public void ThrowsWhenAssigningHeaderContentLengthToNonNumericValue(string contentLength)
```

Comment:
It doesn't look like BadContentLengths contains a negative value. I think we should include that. It would be nice if BadRequestIfContentLengthInvalid also included a test case for negative values.